src/Parsers/ASTDeleteQuery.cpp

```diff
@@ -45,6 +45,12 @@ void ASTDeleteQuery::formatQueryImpl(const FormatSettings & settings, FormatStat
 
     settings.ostr << (settings.hilite ? hilite_keyword : "") << " WHERE " << (settings.hilite ? hilite_none : "");
     predicate->formatImpl(settings, state, frame);
+
+    if (settings_ast)
+    {
+        settings.ostr << (settings.hilite ? hilite_keyword : "") << settings.nl_or_ws << "SETTINGS " << (settings.hilite ? hilite_none : "");
```

Comment:
```suggestion
        settings.ostr << (settings.hilite ? hilite_keyword : "") <<  " SETTINGS " << (settings.hilite ? hilite_none : "");
```
I don't understand but it's the only difference from other clauses.